Added Shadow Pillbug Mesh, Textures, Materials, and Animation & Shado…

red3183 opened the pull request:
…w Army Ant Mesh, Textures

Imported Shadow Pillbug Mesh, Textures, Materials, and Animation. Created Shadow Pillbug Prefab at 40 scale.
Shadow Army Ant Mesh and Textures, no Animations yet.

red3183 committed:
Added Shadow Pillbug Mesh, Textures, Materials, and Animation & Shadow Army Ant Mesh, Textures

Imported Shadow Pillbug Mesh, Textures, Materials, and Animation. Created Shadow Pillbug Prefab at 40 scale.
Shadow Army Ant Mesh and Textures, no Animations yet.

TriOmega reviewed (changes requested):
- Shadow pillbug animation controller doesn't need spaces
- Textures are missing texture type suffix (E.g T_ShadowArmyAntWaist_01**_C**)
- ShadowArmyAntWaist texture is labeled as a material
- Shadow Army Ant Thorax_TXTR needs naming convention
- Not necessarily a required change, more like a point of discussion or suggestion: The materials and textures labeled Section 2-4, it would make sense for these to be like M_Section_01, M_Section_02 if there were a section 1? Feel free to message me about this or keep it as is
- Shadow Army Ant .fbx file needs correct naming covnetion (SM_ShadowArmyAnt_01)
- Message me when you can about what the VisualScripting files/folders are about

red3183 committed:
Updates to the last commit with incorrect naming of a few files.

Updates to the last commit with incorrect naming of a few files.
I don't know what the VisualScripting files/folders is about either.

TriOmega committed:
Organization and merge conflict hot fix

- Removed Visual Scripting folder
- Renamed 2 textures
- Updated ArtPreview prefab with PillBugRigged
-
